core/testdata/TestMonitor.json

```diff
@@ -0,0 +1,31 @@
+{
+	"description": "TestMonitor",
+	"directory": {
```

Comment:
This test, as well as `TestListHistory` and a couple of others, has no RPCs. Is this expected, and they are still different?

Reply:
This is expected. 

At the moment they are not generating RPCs because the individual tests are not generating them.
What I mean to say is that, at the moment, the RPCs are not being collected in a consistent way.  
They are being synthetically synthesized rather than properly recorded with an interceptor. 
This is because we are still injecting client state into the `Action`.  When I remove that, I can properly capture all the RPCs and these files won't be empty.